doc/INSTALL.md

```diff
@@ -259,12 +259,12 @@ Clone lightning:
 
 Configure Python 3.x & get mako:
 
-    $ pyenv local 3.7.4
+    $ pyenv local 3.7.12
     $ pip install mako
```

Comment:
maybe we can add here the mistune dependencies
```suggestion
    $ pip install mako mistune==0.8,4
```

Good suggestion, thanks !

Reply:
Did you mean for that to be `0.8,4` rather than `0.8.4`?

Reply:
ops! my bad, yes the syntax correct is with dots, like the following one.
```suggestion
    $ pip install mako mistune==0.8.4
```

In the doc for Unix system, we have already the same type of command, you can also double-check in the doc.

Sorry for my mistake

Reply:
Let's use the requirements.lock file rather than specifying individual dependencies with versions. Otherwise we'll have to touch these docs every time a dependency changes.

Reply:
Good idea!